src/lib/helpers/config.helper.spec.ts

```diff
@@ -8,5 +8,11 @@ describe('Config Helpper', () => {
     expect(LSP3_UP_KEYS.UNIVERSAL_RECEIVER_DELEGATE_KEY).toEqual(
       '0x0cfc51aec37c55a4d0b1a65c6255c4bf2fbdf6277f3cc0730c45b828b6db8b47'
     );
+    expect(LSP4_KEYS.LSP4_CREATORS_ARRAY).toEqual(
+      '0x114bd03b3a46d48759680d81ebb2b414fda7d030a7105a851867accf1c2352e7'
```

Comment:
Maybe in these tests, instead of comparing with hardcoded literals, it would be better to tests against the keys defined `constants.ts` file in the `@lukso/lsp-smart-contracts`? 🙂 
https://github.com/lukso-network/lsp-smart-contracts/blob/cd29415e8a5731ada3a45dce815f64208491fe4d/constants.ts#L97

Reply:
Ah cool!